### ISSUE TITLE
train/test ratio

### ISSUE THREAD

**Issue by aliosia** [camel007/caffe-moon#6]
Hi, 
What ratio did you use for separating train and test data?
I want to be sure that the current model is not trained on the whole CelebA dataset and the reported result is on test set, so there is no overfitting concern.
Thanks

**Comment by camel007** [comment]
As the database suggest
